lib/ansible/modules/source_control/github_issue.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python
+# (c) 2017, Abhijeet Kasurde <akasurde@redhat.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+module: github_issue
+short_description: View GitHub issue.
+description:
+    - View GitHub issue for a given repository.
+version_added: "2.4"
+options:
+  repo:
+    description:
+      - Name of repository from which issue needs to be retrived.
+    required: true
+    default: none
+  organization:
+    description:
+      - Name of Github organization in which repository is hosted.
+    required: true
+    default: none
+  issue:
+    description:
+      - issue number for which information is required.
+    default: none
+    required: true
+  action:
+    description:
+        - Get various details about issue depending upon action specified
+    default: 'get_status'
+    required: false
```

Comment:
`action=dict(required=True, choices=['get_status']),`

So this should be `required: false`